components/validation/src/main/java/org/wso2/carbon/identity/x509Certificate/validation/CertificateValidationUtil.java

```diff
@@ -168,6 +172,23 @@ public static void addDefaultValidationConfigInRegistry(String tenantDomain) {
         }
     }
 
+    public static void loadCRLDownloadTimeoutFromConfig() {
```

Comment:
Let's add the method description

Reply:
Addressed with cd6d88cb9b11a4d9e6d6fc644f9ba877ce1bb180